posix.mak

```diff
@@ -497,24 +490,11 @@ $(foreach p,$(SRC_DOCUMENTABLES),$(eval \
 $(DOC_OUTPUT_DIR)/$(call D2HTML,$p) : $p $(STDDOC) ;\
   $(DDOC) project.ddoc $(STDDOC) -Df$$@ $$<))
 
-html : $(DOC_OUTPUT_DIR)/. $(HTMLS) $(STYLECSS_TGT)
-
-allmod :
-	@echo $(SRC_DOCUMENTABLES)
-
-rsync-prerelease : html
-	rsync -avz $(DOC_OUTPUT_DIR)/ d-programming@digitalmars.com:data/phobos-prerelease/
-	rsync -avz $(WEBSITE_DIR)/ d-programming@digitalmars.com:data/phobos-prerelase/
-
-html_consolidated :
-	$(DDOC) -Df$(DOCSRC)/std_consolidated_header.html $(DOCSRC)/std_consolidated_header.dd
-	$(DDOC) -Df$(DOCSRC)/std_consolidated_footer.html $(DOCSRC)/std_consolidated_footer.dd
-	$(MAKE) -f $(MAKEFILE) DOC_OUTPUT_DIR=$(BIGDOC_OUTPUT_DIR) STDDOC=$(BIGSTDDOC) html -j 8
-	cat $(DOCSRC)/std_consolidated_header.html $(BIGHTMLS)	\
-	$(DOCSRC)/std_consolidated_footer.html > $(DOC_OUTPUT_DIR)/std_consolidated.html
+# this target is called by dlang.org
+html : $(DOC_OUTPUT_DIR)/. $(HTMLS)
 
-changelog.html: changelog.dd
```

Comment:
We use a changelog folder nowadays. `changelog.dd` has been removed more than a year ago.